lib/puppet/provider/tacacs_server/cisco_ios.rb

```diff
@@ -182,7 +187,30 @@ def create(context, name, should)
       end
     end
 
-    def canonicalize(_context, resources)
+    def canonicalize(context, resources)
```

Comment:
This seems to be the last major thing here.

Same deal as other `canonicalize` fixes, this should go to the `get` function, returning better data instead.